css/style.css

```
@@ -513,22 +544,252 @@ ol.features-list {
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.infrastructure-fieldset {
+  margin-right: 118px;
+}
+
+.cost-fieldset-wrapper {
+  width: 30%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.filter-list {
+  list-style: none;
+  margin: 0;
+  margin-left: 40px;
+  padding: 0;
+}
+
+.filter-label {
+  position: relative;
+  display: block;
+  cursor: pointer;
+  user-select: none;
+  margin-bottom: 26px
+}
+
+.filter-input-checkbox + .filter-label::before {
+  content: "";
+  position: absolute;
+  left: -41px;
+  width: 27px;
+  height: 23px;
+  background-image: url("../img/svg-icon/checkbox-off.svg");
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  background-size: contain;
+}
+
+.filter-input-checkbox:checked + .filter-label::before {
+  content: "";
+  position: absolute;
+  left: -41px;
+  width: 27px;
+  height: 23px;
+  background-image: url("../img/svg-icon/checkbox-on.svg");
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  background-size: contain;
+}
+
+.filter-input-checkbox:disabled + .filter-label::before {
+  content: "";
+  position: absolute;
+  left: -41px;
+  width: 27px;
+  height: 23px;
+  background-image: url("../img/svg-icon/checkbox-off-block.svg");
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  background-size: contain;
+}
+
+.filter-input-checkbox:checked:disabled + .filter-label::before {
+  content: "";
+  position: absolute;
+  left: -41px;
+  width: 27px;
+  height: 23px;
+  background-image: url("../img/svg-icon/checkbox-on-block.svg");
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  background-size: contain;
+}
+
+.price-controls {
+  display: flex;
+  justify-content: flex-start;
+  align-items: stretch;
+  margin-bottom: 19px;
+  border: 2px solid var(--white-transparent);
+  padding: 6px 5px 6px 7px;
+  width: 96%;
+}
+
+.max-price {
+  padding-left: 18px;
+}
+
+.price-input {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.price-controls-input {
+  text-align: center;
+  border: none;
+  background-color: var(--white-transparent-max);
+  color: var(--white);
+  padding: 0;
+  text-transform: uppercase;
+  width: 100%;
+}
+
+.price-controls-input::placeholder {
+  opacity: 1;
+}
+
+.price-controls::after {
+  content: "";
+  position: relative;
+  left: -149px;
+  border: 1px solid var(--white);
+}
+
+.scale {
+  height: 2px;
+  background-color: var(--white-transparent);
+  width: 101%;
+}
+
+.bar {
+  height: 2px;
+  background-color: var(--white);
+  width: 82%;
+}
+
+.range-controls {
+  position: relative;
+  margin-bottom: 20px;
+}
+
+.toggle {
+  display: block;
+  position: absolute;
+  top: -8px;
+  left: 0;
+  width: 4px;
+  height: 4px;
+  padding: 0;
+  border: 7px solid var(--white);
+  background-color: var(--grey-toggle);
+  border-radius: 50%;
+  box-shadow: 0 2px 1px var(--grey-toggle);
+}
+
+.toggle-max {
+  left: 253px;
+}
+
+.toogle:hover {
```

```suggestion
.toggle:hover {
```

Опечатка. Поэтому и не работало 😬